layouts/fastly/helix.vcl

```diff
@@ -397,10 +395,24 @@ sub vcl_recv {
       set var.path = regsuball(var.path, "/+", "/");
       set req.url = "/" + var.owner + "/" + var.repo + "/" + var.ref + var.path + "?" + req.url.qs;
     } else {
+      # get (strain-specific) parameter whitelist
+      include "params.vcl";
+
+
       set var.path = var.dir + "/" + var.name + ".md";
       set var.path = regsuball(var.path, "/+", "/");
       # Invoke OpenWhisk
-      set req.url = "/api/v1/web" + var.action + "?owner=" + var.owner + "&repo=" + var.repo + "&ref=" + var.ref + "&path=" + var.path + "&selector=" + var.selector + "&extension=" + req.url.ext + "&branch=" + var.branch + "&strain=" + var.strain + "&GITHUB_KEY=" + table.lookup(secrets, "GITHUB_TOKEN");
+      set req.url = "/api/v1/web" + var.action + 
+        "?owner=" + var.owner + 
+        "&repo=" + var.repo + 
+        "&ref=" + var.ref + 
+        "&path=" + var.path + 
+        "&selector=" + var.selector + 
+        "&extension=" + req.url.ext + 
+        "&branch=" + var.branch + 
+        "&strain=" + var.strain + 
+        "&GITHUB_KEY=" + table.lookup(secrets, "GITHUB_TOKEN") +
```

Comment:
I'm really not happy to pass the token as request parameter. can we send a POST instead?

Reply:
We can't, but it isn't too much of a problem, as we can pass the token as a default parameter to the action. The `GITHUB_TOKEN` wasn't used anyway, so I've removed the offending line.